lib/single-new-eks-fargate-opensource-observability-pattern/index.ts

```diff
@@ -122,8 +122,7 @@ export default class SingleNewEksFargateOpenSourceObservabilityConstruct {
         Reflect.defineMetadata("ordered", true, blueprints.addons.GrafanaOperatorAddon);
         const addOns: Array<blueprints.ClusterAddOn> = [
             new blueprints.addons.VpcCniAddOn(),
-            new blueprints.addons.CoreDnsAddOn({
-                version: "v1.10.1-eksbuild.6",
+            new blueprints.addons.CoreDnsAddOn("v1.10.1-eksbuild.6", {
```

Comment:
version can be populated with auto or can be removed!

Reply:
done